scripts/output/vaccinations/main_data/Azerbaijan.csv

```diff
@@ -311,5 +311,5 @@ Azerbaijan,2022-02-14,"Oxford/AstraZeneca, Pfizer/BioNTech, Sinovac, Sputnik V",
 Azerbaijan,2022-02-15,"Oxford/AstraZeneca, Pfizer/BioNTech, Sinovac, Sputnik V",https://koronavirusinfo.az/storage/app/MvN7udDQpfRpye6gH9GF7aIwLueGHk0PTRK18Rvg.pdf,12549963,5270596,4766430,2512937,
 Azerbaijan,2022-02-16,"Oxford/AstraZeneca, Pfizer/BioNTech, Sinovac, Sputnik V",https://koronavirusinfo.az/storage/app/9sWKkwHJeU9OdgY7G2YVU8Ux0MmAsUuc5WuhqRpD.pdf,12627370,5273971,4768299,2585100,
 Azerbaijan,2022-02-17,"Oxford/AstraZeneca, Pfizer/BioNTech, Sinovac, Sputnik V",https://koronavirusinfo.az/storage/app/j5Ans4opJcxDqbdaOJnFYGnwYaxs5mAP9iiUeQLT.pdf,12720225,5280393,4772019,,
-Azerbaijan,2022-03-06,"Oxford/AstraZeneca, Pfizer/BioNTech, Sinovac, Sputnik V",https://koronavirusinfo.az/storage/app/Q20cnFjBygDuERm7dAZPJN0dzPYQ7AsAlNvJtWqm.pdf,13159751,5308772,5008914,2842065,
+Azerbaijan,2022-03-06,"Oxford/AstraZeneca, Pfizer/BioNTech, Sinovac, Sputnik V",https://koronavirusinfo.az/storage/app/Q20cnFjBygDuERm7dAZPJN0dzPYQ7AsAlNvJtWqm.pdf,13159751,5308772,,2842065,
```

Comment:
this data point can be deleted all together if not necessary. otherwise I can track it down and find the values.

Reply:
It is OK to have it incomplete, so we have some of the metrics at least.

Reply:
The data is available here: https://www.marja.az/public/84515/azerbaycanda-koronavirus-ile-bagli-son-veziyyet-aciqlandi

Reply:
4795324 , 213590 respectively.